src/main/java/org/skriptlang/skript/lang/converter/EventConverter.java

```diff
@@ -0,0 +1,22 @@
+package org.skriptlang.skript.lang.converter;
```

Comment:
I would put this within the Bukkit package, something like `org.skriptlang.skript.bukkit.event`

Reply:
That is a good call. I didnt put much thought into which package to put it in, but that makes more sense.
Ill make that change.